spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -5,33 +5,52 @@
                             password: "pass",
                             name: "jorge")
   }
-  let(:create_item) { LoanRequest.create(title: "Farm Tools",
-                                         description: "help out with the farm tools",
-                                         amount: "$100.00",
-                                         requested_by_date: "2015-06-01",
-                                         repayment_begin_date: "2015-12-01",
-                                         repayment_rate: "monthly",
-                                         user_id: user.id)
+  let (:user2) { User.create(email: "jeff@example.com",
+                             password: "pass",
+                             name: "jeff",
+                             role: 1)
   }
+  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.